dev-scripts/translate-source/scan-translations.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+#
+# The script must be run from within a running lagoon cli container and expects
+# the root of the dpl-cms repository to be available at /app.
+# The script is used to scan the translations of the custom modules and themes
+# and is dependent on the potion module.
+
+set -eo pipefail
+
+LANGUAGE="$1"
+PO_DIR="$2"
+
+if [[ -z "${LANGUAGE}" ]]; then
+	echo "usage: $0 <LANGUAGE> <PO_DIR>" >&2
+	exit 1
+fi
+if [[ -z "${PO_DIR}" ]]; then
+	echo "usage: $0 <LANGUAGE> <PO_DIR>" >&2
+	exit 1
+fi
+
+# Get the directory of the script
+THIS_DIR=$(dirname "$0")
+
+
+# First import translations from custom modules.
+drush potion:generate $LANGUAGE modules/custom/ $PO_DIR --recursive --default-write-mode=override
+
+# Define an array of modules to exclude
+include_modules=()
+while IFS= read -r line; do
+    include_modules+=("$line")
+done < ${THIS_DIR}/scanned_modules.txt
+
+for module in "${include_modules[@]}"; do
+    # Prepend the base path to the directory
+    dir="web/modules/contrib/$module"
+
+    if [ -d "$dir" ]; then
+        echo "Processing $module:"
+        dir=$(echo $dir | sed 's/^web\///')
+        drush potion:generate da $dir profiles/dpl_cms/translations --recursive --default-write-mode=merge
```

Comment:
Should the default-write-mode be "merge" or "override"? I can see that it was set to "override" when all contrib modules was being scanned.

Reply:
The [first operation](https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/925/files#diff-55569087b6a5448bba7209331df93ef27c9f2ac4e2c5388ff1018fa8ada49354R27), that covers the custom modules, **overrides**, which basically means starting on a clean slate.
The following two opererations scans contrib modules and themes and they **merge**, which means they add to the translations from the first operation.
So it is setup as intended.

Reply:
Ah okay, it makes sense. 👍 

However shouldn't this also be the way it was implemented before? From your explanation it seems that in the previous implementation we would always override the the scanning of the contrib modules when starting scanning the custom modules? Maybe I am missing something 😄

Reply:
The former change that added the scanning of contrib modules was accidentally set to **override** which was a mistake.
The idea is that the first operation **overrides** and the rest **merges**.
Does it make sense?

Reply:
Yes that makes sense 👍 Just wasn't sure if the former was on purpose or a mistake 😄